src/roman_datamodels/maker_utils/_common_meta.py

```diff
@@ -481,9 +481,10 @@ def mk_mosaic_meta(**kwargs):
     dict (defined by the wfi_mosaic-1.0.0 schema)
     """
 
-    meta = {}
+    meta = mk_basic_meta(**kwargs)
+    meta["basic"] = mk_mosaic_basic(**kwargs.get("meta", {}))
```

Comment:
```suggestion
    meta["basic"] = mk_mosaic_basic(**kwargs.get("basic", {}))
```

Reply:
Thanks! That was the test issue, fixed now.